src/field_theory/separable.lean

```diff
@@ -291,6 +415,29 @@ begin
   rw [hf2, is_unit_C] at hf1, rw expand_eq_C hp at hf2, rwa [hf2, is_unit_C]
 end
 
+variables {R : Type u} [comm_ring R] [is_domain R]
```

Comment:
Why not keep the original variables line
`variables (R : Type u) [comm_ring R] [is_domain R]`
and replace this line with
`variables {R}`?

Reply:
I didn't think that worked! done:) I assumed it either would tell me `R` was already defined, or overwrite the definition of `R` and remove the typeclasses